File: backend/last-market/src/main/java/edu/ssafy/lastmarket/domain/eneity/Product.java
package edu.ssafy.lastmarket.domain.eneity;


import javax.persistence.*;
import java.time.LocalDateTime;
import java.util.ArrayList;
import java.util.List;

@Entity
public class Product extends BaseEntity{
    @Id
    @GeneratedValue
    @Column(name = "product_id")
    private Long id;
    private String title;
    private String content;
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "dongCode")
    private Location location;
    private DealState dealState;
    @OneToMany(fetch = FetchType.LAZY)
    @JoinColumn(name = "product_image_id")
    private List<ProductImage> images = new ArrayList<>();
    private LocalDateTime liveTime;
    @OneToOne(fetch = FetchType.LAZY)
    private Category category;
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "seller_id")
    private Member seller;
    private Integer favoriteCnt;
    private Long startingPrice;
    private Long instantPrice;
}

File: mobile/LastMarket/app/src/main/java/com/jphr/lastmarket/dto/Product.kt
package com.jphr.lastmarket.dto

import java.time.LocalDateTime

data class Product(
    val createdDateTime: String,
    val dealState: String,
    val favoriteCnt: Long,
    val imgURI: String,
    val instantPrice: Long,
    val isFavorite: Boolean,
    val liveTime: String?,
    val location: String,
    val productId: Long,
    val sellerId: Long,
    val sellerNickname: String,
    val startingPrice: Long,
    val title: String
)